src/sidebar/store/modules/selection.js

```diff
@@ -163,27 +173,38 @@ const setTab = (newTab, oldTab) => {
   };
 };
 
+/**
+ * Return state object representing a reset selection. This
+ * resets user-set filters (but leaves focus mode intact)
+ *
+ * @return {Object}
```

Comment:
Was there a reason for the `@return {Object}` here? Omitting it doesn't cause an issue AFAICS and it will allow TS to do better inference in future.